<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeAxisCombineDialog</class>
 <widget class="QDialog" name="TimeAxisCombineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time-axis Combine Two Nifti Files</string>
  </property>
  <widget class="QLabel" name="label_timeComb_imgData1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>451</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_timeComb_imgData2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>530</y>
     <width>421</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_timeComb_inputs">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_timeComb_UplData1">
        <property name="text">
         <string>Load Data 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_timeComb_UplData2">
        <property name="text">
         <string>Load Data2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_outData">
        <property name="text">
         <string>Load Output Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_timeComb_combine">
        <property name="text">
         <string>Combine</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>880</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_timeComb_svaeImgName">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_timeComb_saveImgName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_timeComb_saveImg">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
